src/data/markdown/docs/30 xk6-browser/01 xk6-browser/01 Browser/newcontext--options--.md

```diff
@@ -23,34 +23,34 @@ You can customize the creation of a new browser context using the following opti
+| ✅ | httpCredentials                                                     | object  | Sets the credentials for HTTP authentication using Basic Auth. See: [httpCredentials](#httpcredentials-options).                                                                                                                                                                                                                                                                                          |
+| ✅ | ignoreHTTPSErrors                                                   | boolean | Whether to ignore HTTPS errors that may be caused by invalid certificates. Defaults to `false`.                                                                                                                                                                                                                                                                                          |
+| ✅ | isMobile                                                            | boolean | Whether to simulate a mobile device. Defaults to `false`.                                                                                                                                                                                                                                                                                                                                |
+| ✅ | javaScriptEnabled                                                   | boolean | Whether to activate JavaScript support for the context. Defaults to `true`.                                                                                                                                                                                                                                                                                                              |
+| ✅ | locale                                                              | string  | Specifies the user's locale, such as `en-US`, `tr-TR`, etc.                                                                                                                                                                                                                                                                                                                              |
+| <span title="Not implemented">❌</span> | logger                      | object  | Specifies the logger to use in xk6-browser.                                                                                                                                                                                                                                                                                                                                              |
```

Comment:
As with other non-implemented things, I'd remove this line, but this appears to be a way of configuring the logger, so "Specifies the logger" doesn't sound right.